decidim-proposals/app/views/decidim/proposals/admin/proposals/_form.html.erb

```diff
@@ -2,33 +2,11 @@
   <div class="card pt-4">
     <div class="card-section">
       <div class="row column pb-4">
-        <%= form.translated :text_field, :title, autofocus: true, class: "js-hashtags", hashtaggable: true, aria: { label: :title } %>
+        <%= form.translated :text_field, :title, autofocus: true, class: "js-hashtags", aria: { label: :title } %>
```

Comment:
```suggestion
        <%= form.translated :text_field, :title, autofocus: true, aria: { label: :title } %>
```

Can we just delete class entirely for this @andreslucena?

Reply:
Yes. let's remove this from all the modules (even `decidim-core`). I understand that even the user story definition refers to proposals, the same arguments can be made to other modules.

We need to remove both `js-hashtags` and `hashtaggable` keyword argument/class.